plugins/txn_test_gen_plugin/txn_test_gen_plugin.cpp

```diff
@@ -127,14 +127,14 @@ struct txn_test_gen_plugin_impl {
          if(ec)
             return;
          try {
-            send_transaction();
+            this->send_transaction();
```

Comment:
we are pretty sure that the situation causing the need for the explicit `this->` was due to the use of `auto ec` at the time this change was made to master.  Can we reconfirm that this is necessary now that we are explicitly typing the input parameter?  

refer to: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=67274

Reply:
Ok. I removed this.